src/components/Calculator.js

```diff
@@ -0,0 +1,28 @@
+function Calc() {
```

Comment:
[README] Kindly use the `Calculator` as the name of the function. Using the same name as the file name is a good practice. 

[REQUIRED] It is a requirement for your calculator component to have at least one child component. Kindly do this